pkg/sql/internal.go

```diff
@@ -93,14 +93,27 @@ func MakeInternalExecutor(
 }
 
 // SetSessionData binds the session variables that will be used by queries
-// performed through this executor from now on.
+// performed through this executor from now on. sessionData can be nil. If
+// sessionData is not nil, it will be populated with the minimum values needed
+// in order to not crash.
 //
 // SetSessionData cannot be called concurently with query execution.
 func (ie *InternalExecutor) SetSessionData(sessionData *sessiondata.SessionData) {
-	ie.s.populateMinimalSessionData(sessionData)
+	if sessionData != nil {
```

Comment:
I don't think this guy should accept a nil. I hope this becomes moot if it's true that `GetSessionData()` always returns `nil` in `Server.Start`.

Reply:
Removed.